doc/quick-reference-guide.md

```diff
@@ -0,0 +1,185 @@
++ `defcfg`: configuration rules (input and output devices, etc).
+
++ `input` and `output`:
+	- For Linux:
+       ```
+       input  (device-file "/dev/input/by-id/usb-04d9_daskeyboard-event-kbd")
+       output (uinput-sink "My KMonad output")
+       ```
+	- For Windows:
+       ```
+       input  (low-level-hook)
+       output (send-event-sink)
+       ```
+	- For MacOS:
+		```
+		input  (iokit-name "my-keyboard-product-string")
+		output (kext)
+		```
++ `fallthrough` (boolean, defaults to `false`): allow all keys, even if not defined
+in defsrc.
+
++ `allow-cmd` (boolean, defaults to `false`): allow commands to be run as keys (very
+dangerous).
+
++ `cmp-seq` (key, defaults to `RightAlt`): compose key for Unicode input (Linux X11
+specific).
+
+	+ `cmp-seq-delay` (number): delay between each pressed key in a compose-key
+sequence
+```
+(defcfg
+  ;; For Linux
+  input  (device-file "/dev/input/by-id/usb-04d9_daskeyboard-event-kbd")
+  output (uinput-sink "My KMonad output" "/run/current-system/sw/bin/sleep 1 && /run/current-system/sw/bin/setxkbmap -option compose:ralt")
+
+  cmp-seq ralt
+  cmp-seq-delay 5
+
+  fallthrough true
+
+  allow-cmd true
+)
+```
```

Comment:
Is this now an example of the whole thing up to this point?  I would preface this by a few lines of text, this is kind of hard to read as it stands

Reply:
I thought that adding an example at the end of how the whole block should look like would be helpful. People may think they are separated parts and I think using an example is easier and clearer than trying to put the idea into words.